### PR TITLE
feat(parental-leave): plug all select and radio controller and refactor stateful logic

### PR DESCRIPTION
**Pull request by jeremybarbet**
## What

- Bunch of fields were not plugged to be editable.
- Refactor stateful logic to make it on all fields retrieved from answers by default
- Fix mobile layout

### CODE REVIEW

File: libs/application/templates/parental-leave/src/hooks/useEstimatedPayments.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            ratio: 100,
+            approved: true,
+            paid: true,
+          },
+        ],
+      },
+    },
+  })
+  */
+}
```
Comment (saevarma) [inline libs/application/templates/parental-leave/src/hooks/useEstimatedPayments.ts:24]: Maybe this file should be removed completely while not in use.
Than added later when this is implemented again.

**Review by saevarma — CHANGES_REQUESTED** [review]
Looks good, just one minor feedback.

File: libs/application/templates/parental-leave/src/hooks/useEstimatedPayments.ts (hunk 1 above)
Comment (jeremybarbet) [reply libs/application/templates/parental-leave/src/hooks/useEstimatedPayments.ts:24]: Yes I was not sure, but you are right, and the API might might be different as well so I will do that 👍

**Review by saevarma — APPROVED** [review]
Nice!

File: libs/application/templates/parental-leave/src/hooks/useStatefulAnswers.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+} from './useApplicationAnswers'
+
+export const useStatefulAnswers = (
+  application: Application,
+): [ApplicationAnswers, Dispatch<SetStateAction<ApplicationAnswers>>] => {
+  const answers = useApplicationAnswers(application)
+  const [stateful, setStateful] = useState(answers)
+
+  return [stateful, setStateful]
+}
```
Comment (baering) [inline libs/application/templates/parental-leave/src/hooks/useStatefulAnswers.ts:16]: Not used anywhere?

**Review by baering — COMMENTED** [review]
LGTM.

`useStatefulAnswers` does not seem to be used anywhere.

File: libs/application/templates/parental-leave/src/hooks/useStatefulAnswers.ts (hunk 2 above)
Comment (jeremybarbet) [reply libs/application/templates/parental-leave/src/hooks/useStatefulAnswers.ts:16]: Yes, it's used in the big file diff that's being minimized :D 
Screenshot 2021-05-31 at 10 15 06

**Review by madebynoam — APPROVED** [review]

File: libs/application/templates/parental-leave/src/hooks/useStatefulAnswers.ts (hunk 2 above)
Comment (baering) [reply libs/application/templates/parental-leave/src/hooks/useStatefulAnswers.ts:16]: 🙈

**Pull request merged**